ansible-ipi-install/README.md

```diff
@@ -279,6 +279,11 @@ dnsvip=""
 # Network Type (OpenShiftSDN or OVNKubernetes). Playbook defaults to OVNKubernetes.
 # Uncomment below for OpenShiftSDN
 #network_type="OpenShiftSDN"
+# (Optional) An URL to override the default operating system image for the bootstrap node.
+# The URL must contain a sha256 hash of the image.
+# See https://github.com/openshift/installer/blob/master/docs/user/metal/customization_ipi.md
+#   Example https://mirror.example.com/images/qemu.qcow2.gz?sha256=a07bd...
+#bootstraposimage=""
```

Comment:
Edu don't kill me lol -- but to be consistent with the variable used in the documentation, can we change it from 'bootstraposimage' to 'bootstrapOSImage' ?